seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter/src/main/java/org/apache/seatunnel/core/starter/flink/FlinkStarter.java

```diff
@@ -90,7 +90,7 @@ public List<String> buildCommands() {
         flinkCommandArgs.getVariables().stream()
                 .filter(Objects::nonNull)
                 .map(String::trim)
-                .forEach(variable -> command.add("-D" + variable));
+                .forEach(variable -> command.add("-i " + variable));
```

Comment:
https://github.com/apache/seatunnel/blob/e5df7f00f8e21edcc5168c4f3dedfc7f85a4a150/seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/command/AbstractCommandArgs.java#L48
because we use `-i ` as the variables parameter name. if we use `-D` then we can't get the variables

Reply:
Got it.